ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -56,6 +57,10 @@
       Pattern.compile("\\s*UNSET\\s+'((?:[^']*|(?:''))*)'\\s*;\\s*");
   private static final Pattern DROP_CONNECTOR =
       Pattern.compile("\\s*DROP\\s+CONNECTOR\\s+(.*?)\\s*;\\s*");
+  private static final Pattern DEFINE_VARIABLE = Pattern.compile(
+      "\\s*DEFINE\\s+([a-zA-Z_][a-zA-Z0-9_@]*)\\s*=\\s*'((?:[^']*|(?:''))*)'\\s*;\\s*");
```

Comment:
Just checking: DEFINE statements require that the values be strings, even though the java client and HTTP endpoints allow non-string values, right? It looks like the values are converted into strings by `SessionProperties` anyway. Is there a reason we didn't have the HTTP endpoints require string values to keep things simpler/more consistent? It's worth considering whether the java client should only accept string values as well.

Reply:
It follows the same pattern as `SET`, which also requires strings in the statement but can be passed as object parameters. I can see two reasons this could be convenient:

1. Users using the endpoints don't need to treat `sessionProperties` and `sessionVariables` differently
2. Client users can make requests like `client.define("name", 5);` or `client.define("name", false);` without having to convert them to strings.

These are pretty minor reasons though. Swapping everything to strings is a pretty easy change.

Reply:
That makes sense. Let's leave this as is!